.github/workflows/db_production_flask_migration_manual.yaml

```diff
@@ -1,7 +1,19 @@
-name: Production - Manual DB Downgrade
+name: Production - Manual DB Flask Command
 
 on:
   workflow_dispatch:
+    inputs:
+      password:
+        description: "Enter the password - can be found here: https://start.1password.com/open/i?a=BZ4RYZZICJEKNOF4Z6ASRX2XXI&v=ppnxsriom3alsxj4ogikyjxlzi&i=i5kfypu7grhf5lc2bu4mi4yque&h=cds-snc.1password.ca"
+        required: true
+        default: ""
+      flask_command:
+        description: "Select the Flask command to run"
+        required: true
+        type: choice
+        options:
+          - upgrade
+          - downgrade
```

Comment:
That should be sufficient for us at this point in time 👍 If we need to pass in versions, that can be another iterative improvement.